javalin-graphql/src/main/java/io/javalin/plugin/graphql/GraphQLPlugin.kt

```diff
@@ -0,0 +1,29 @@
+package io.javalin.plugin.graphql
+
+import io.javalin.Javalin
+import io.javalin.core.plugin.Plugin
+import io.javalin.core.plugin.PluginLifecycleInit
+
+class GraphQLPlugin(private val options: GraphQLOptions) : Plugin, PluginLifecycleInit {
+
+    lateinit var graphQLHandler: GraphQLHandler
+
+    override fun init(app: Javalin) {
+        graphQLHandler = GraphQLHandler(options)
+    }
+
+    override fun apply(app: Javalin) {
+        app.get(options.path) {
+            it.contentType("text/html; charset=UTF-8")
+                    .result(
+                            GraphQLPlugin::class.java.getResourceAsStream("graphqli/index.html")
```

Comment:
Graphiql could also be moved into a separate module, because there are several alternatives.

Reply:
My idea was to have a "complete" graphql environment. Another module could be created, but it is a file. What do you propose?

Reply:
I don't know about the alternatives, but I think that `javalin-graphql` should contain everything required to start using graphql (so probably what you also mean with "complete" environment). 

Javalin itself is not very opinionated, but I think it's good if the plugins/modules are.

Reply:
Maybe, it's necessary create some handler for auth.